### PR TITLE
add redux logger

### PR DESCRIPTION
**Pull request by plandem**

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ export default function configureStore ({ initialState = {}, history }) {
 
   // Compose final middleware and use devtools in debug environment
   let middleware = applyMiddleware(thunk, routerMiddleware)
-  if (__DEBUG__) middleware = withDevTools(middleware)
+  if (__DEBUG__) {
+    var createLogger = require('redux-logger');
```
Comment (thangngoc89) [inline src/redux/configureStore.js]: Why `var` ?

**Comment by plandem** [comment]
previous background :) it can be const :)

**Comment by thangngoc89** [comment]
I think it should be `require('redux-logger').default`

**Comment by davezuko** [comment]
CI has been broken for a few weeks now, so going to close this. If you'd like to resubmit with a fixed build and an explanation as to why you think this would be a good addition to the starter kit, I'd be happy to reconsider. Thanks!

**Comment by plandem** [comment]
Sorry, I'm not using react-router anymore, so had to create own starter kit.

**Pull request closed without merging**